tests-behat/bootstrap/Context.php

```diff
@@ -482,41 +482,41 @@ public function scopeBuilderBoolRule($name, $value)
     }
 
     /**
-     * @Then /^I check if word from data scope match$/
+     * @Then /^I check if text in "([^"]*)" match text in "([^"]*)"/
      */
-    public function iCheckIfWordMatch()
+    public function compareElementText($compareSelector, $compareToSelector)
     {
-        $pageWordContainer = $this->getSession()->getPage()->find('css', '.atk-expected-word-result .content');
-        if (!$pageWordContainer) {
-            throw new \Exception('Unable to find expected word in page.');
+        $compareContainer = $this->getSession()->getPage()->find('css', $compareSelector);
+        if (!$compareContainer) {
+            throw new \Exception('Unable to find compare container: ' . $compareSelector);
         }
 
-        $expectedWord = $pageWordContainer->getText();
+        $expectedText = $compareContainer->getText();
 
-        $wordResponseContainer = $this->getSession()->getPage()->find('css', '.atk-callback-response .content');
-        if (!$wordResponseContainer) {
-            throw new \Exception('Unable to find response word in page.');
+        $compareToContainer = $this->getSession()->getPage()->find('css', $compareToSelector);
+        if (!$compareToContainer) {
+            throw new \Exception('Unable to find compare to container: ' . $compareToSelector);
         }
 
-        $responseWord = $wordResponseContainer->getText();
+        $compareToText = $compareToContainer->getText();
 
-        if (preg_replace('/\s*/m', '', $expectedWord) !== preg_replace('/\s*/m', '', $responseWord)) {
-            throw new \Exception('Data word does not match: ' . $responseWord . ' expected: ' . $expectedWord);
+        if (preg_replace('~\s*~', '', $expectedText) !== preg_replace('~\s*~', '', $compareToText)) {
```

Comment:
When expected test is generated, when it does not match binary? I think, we do not need this replace... 2x

Reply:
done

Reply:
also for `compareInputValueToElementText`

Reply:
test fail without preg_replace for input...

Reply:
then we expect wrong input ;-)

Reply:
Both, $expected and $input-getValue(), contains space char. $expected for readability and $get->input() for "matches regular expression" so we need to remove space in both place for test to pass.